README.md

```diff
@@ -1,36 +1,95 @@
+
+	// Start scanning.
+	println("scanning...")
+	err := adapter.Scan(func(adapter *bluetooth.Adapter, device bluetooth.ScanResult) {
+		println("found device:", device.Address.String(), device.RSSI, device.LocalName())
+	})
+	must("start scan", err)
+}
+
+func must(action string, err error) {
+	if err != nil {
+		panic("failed to " + action + ": " + err.Error())
+	}
+}
+```
+
+## Current support
+
+|                                  | Linux              | macOS              | Windows            | Nordic Semi        |
 | -------------------------------- | ------------------ | ------------------ | ------------------ | ------------------ |
-| API used                         | WinRT              | BlueZ (over D-Bus) | SoftDevice         | CoreBluetooth      |
+| API used                         | BlueZ              | CoreBluetooth      | WinRT              | SoftDevice         |
 | Scanning                         | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
-| Connect to peripheral            | :x:                | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
-| Write peripheral characteristics | :x:                | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
-| Receive notifications            | :x:                | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
-| Advertisement                    | :x:                | :heavy_check_mark: | :heavy_check_mark: | :x:                |
-| Local services                   | :x:                | :heavy_check_mark: | :heavy_check_mark: | :x:                |
-| Local characteristics            | :x:                | :heavy_check_mark: | :heavy_check_mark: | :x:                |
-| Send notifications               | :x:                | :heavy_check_mark: | :heavy_check_mark: | :x:                |
+| Connect to peripheral            | :heavy_check_mark: | :heavy_check_mark: | :x:                | :heavy_check_mark: |
+| Write peripheral characteristics | :heavy_check_mark: | :heavy_check_mark: | :x:                | :heavy_check_mark: |
+| Receive notifications            | :heavy_check_mark: | :heavy_check_mark: | :x:                | :heavy_check_mark: |
+| Advertisement                    | :heavy_check_mark: | :x:                | :x:                | :heavy_check_mark: |
+| Local services                   | :heavy_check_mark: | :x:                | :x:                | :heavy_check_mark: |
+| Local characteristics            | :heavy_check_mark: | :x:                | :x:                | :heavy_check_mark: |
+| Send notifications               | :heavy_check_mark: | :x:                | :x:                | :heavy_check_mark: |
+
+## Linux
+
+The current support for Linux uses BlueZ via the D-Bus interface. It should work with most distros that support BlueZ such as Ubuntu, Debian, Fedora, and Arch Linux, among others. Linux can be used both as a BLE central as well as BLE peripheral.
```

Comment:
It might be worth pointing out that BlueZ shouldn't be too old, BLE support in BlueZ has only become stable in the last 2-3 years or so.

Reply:
Added some additional details.